lib/missing_transaction.go

```diff
@@ -0,0 +1,37 @@
+package sebak
+
+import "encoding/json"
+
+type MissingTransactions struct {
```

Comment:
It is just a map of transactions.
do we need this structure?

Reply:
@kfangw yep. this for just only `Get Transaction from proposer`. 
if you think that this is unnecessary, I will change it. 
what do you think of this?